MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Use UpDown Button in .NET MAUI NumericEntry | Syncfusion
+description: Learn here all about how to use UpDown Button (SpinButton) in Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control and more.
+platform: MAUI
+control:  SfNumericEntry
+documentation: ug
+---
+
+# UpDown Button in .NET MAUI NumericEntry
+
+This section describes how to change value in the `NumericEntry` control using keys, mouse scrolling and the up-down button.
+
+## Increase or decrease value
+
+You can increment or decrement the value in the `NumericEntry` control by using **UpArrow**, **DownArrow**, **PageUp** and **PageDown** keys. You can change the increment or decrement value when Arrow keys are pressed, using the SmallChange property and for Page keys using the LargeChange property. By default, the value of `SmallChange` property is **1** and `LargeChange` property  is **10**.
+
+N> The value in `NumericEntry` can also be changed by mouse scrolling. The mouse scrolling increases or decreases the value based on the `SmallChange` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry HorizontalOptions="Center"
+                     VerticalOptions="Center" 
+                     SmallChange="5"
+                     Value="10"
+                     LargeChange="10" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry SfNumericEntry= new SfNumericEntry();
+SfNumericEntry.Placeholder = "Enter input here...";
+sfNumericEntry.HorizontalOptions = LayoutOptions.Center;
+sfNumericEntry.VerticalOptions = LayoutOptions.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry Watermark Text](GettingStarted_images/valuechange-bykeys.gif)
+
+## UpDown button placement
+
+You can increase or decrease the value of the `NumericEntry` control using the up-down button. By default, the value of `UpDownPlacementMode` property is **Hidden**. You can change the up-down button position by assigning the value `UpDownPlacementMode` property as **Inline**.
```

Comment:
You can increase or decrease the value of the `NumericEntry` control using the up-down button. By default, the value of the `UpDownPlacementMode` property is **Hidden**. You can change the up-down button position by assigning the value `UpDownPlacementMode` property as **Inline**.

Reply:
The requested content modification has been successfully addressed